File: Miami GP.c
#include<stdio.h>
int main()
{
    int t;
    scanf("%d",&t);
    while(t--)
    {
        int x,y,a;
        scanf("%d%d",&x,&y);
        a=(x*107)/100;
        if(y<=a)
        printf("YES\n");
        else
        printf("NO\n");
    }
}

File: Miami GP.java
import java.util.*;
class Codechef
{
    public static void main(String args[])
    {
        Scanner sc=new Scanner(System.in);
        int t=sc.nextInt();
        while(t-- >0)
        {
            int x=sc.nextInt();
            int y=sc.nextInt();
            if(y<=((x*107)/100))
            System.out.println("YES");
            else
            System.out.println("NO");
        }
    }
}